.gitattributes

```diff
@@ -1,25 +1,2 @@
-# DO NOT CHANGE THIS FILE
-# DO NOT EDIT THE LINE BELOW.
-/.gitattributes merge=gitattributes
-# DO NOT EDIT THE LINE ABOVE.
-#
-# You can of course edit this file, but make sure you understand what you are
-# doing. This file defines a custom filter driver that prevents snapshot test
-# images from being merged into `stable`. Snapshot test images are only
-# valuable in `develop` because they are only intended to help developers
-# identify changes in the appearance of the library.
-#
-# Before you change this file, please carefully consider whether such a change
-# is actually necessary. When you do change this file, it should almost always
-# be done in a dedicated commit directly on the `stable` branch and not part
-# of a release.  If you see this file being changed as part of a release,
-# block the release and work with the releaser to ensure that the change needs
-# to be propagated from the `develop` branch to the `stable` branch. In nearly
-# all cases, it should not be propagated from `develop` to `stable`.
-#
-# If you are a releaser and see this file change and you're not sure why, you
-# might have accidentally skipped [setting the correct
-# driver in your cloned
-# repository](https://github.com/material-components/material-components-ios/blob/develop/contributing/releasing.md#configure-the-merge-strategy-for-gitattributes).
-# If that's the case, please either revert the accidental change manually or
-# restart the release with a fresh clone and the correct driver.
+snapshot_test_goldens/**/*.png filter=lfs diff=lfs merge=lfs -text
```

Comment:
@romoore Is this change expected?

Reply:
No, that should not be merged. It's #8392 showing up again.

Reply:
You can do what I did - manually edit `.gitattributes` in the `release-candidate` branch to restore it. However, you then have to fix it again on `develop` since the fix commit won't be present in `develop`.

Reply:
Got it; is there an automated solution to this problem that we can implement?

Reply:
Hopefully there is, but we'll have to prioritize #8392 to identify what it might be and possibly test it out. If not, it may be easiest to stage the `release-candidate` branch so the On-Call can simply revert that change before performing the merge commit.